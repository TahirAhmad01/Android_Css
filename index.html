<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Android</title>
    <link rel="icon" type="icon" href="img/ico.png">
    <link rel="stylesheet" href="fsm/css/all.css">
    <link rel="stylesheet" href="css/font.css">
    <link rel="stylesheet" href="css/owl.carousel.min.css">
    <link rel="stylesheet" href="css/style.css">
</head>
<body>
    <div class="mobile-freame-size">
        <div class="loader">
            <div class="loader-image">
                <div>
                    <img src="img/oppo_logo.jpg">
                </div>
                <div>
                    <img src="img/powerd_by.png">
                </div>
            </div>
        </div>
        <div class="body-flex">
            <div class="notification-bar">
                <div class="notification_flex">
                    <div class="left-notification">
                        <div class="f_cam">
                            <img src="img/front-cam.png">
                        </div>
                        <div class="notification-time">
                            <text class="hour"></text>:<text class="minute"></text>
                        </div> 
                    </div>
                    <div class="notification-second-panel">
                        <i class="fad fa-signal-alt"></i>
                        <i class="fad fa-signal-3"></i>
                        <i class="far fa-battery-full"></i>
                    </div>
                </div>
            </div>

            <div class="middle_body">
                <div class="hider-div">
                    <div class="mobile-scroll-body">
                        <div class="owl-carousel  owl-theme">
                            <div class="item">
                                <div class="slider-body first-slider-body">
                                    <div class="google-search-bar">
                                        <div class="google-round-search">
                                            <div class="g-logo">
                                                <img src="img/Google__G__Logo.png">
                                            </div>
                                            <div class="google-v-l">
                                                <img src="img/G_voice.png">
                                                <img src="img/G_leans.png">
                                            </div>
                                        </div>
                                    </div>

                                    <div class="body-time-info">
                                        <div class="body-time">
                                            <div>
                                                <text class="hour">51</text>
                                            </div>
                                            <div>
                                                <text class="minute"></text>
                                            </div>
                                        </div>
                                        
                                        <div class="body-date">
                                            <text class="day"></text>, <text class="month"></text> <text class="date"></text>
                                        </div>
                                    </div>
                                </div>
                            </div>

                            <div class="item">
                                <div class="slider-body second-slider-body">
                                    <div class="items-inner">
                                        <div>
                                            <div id="cute">
                                                <img src="img/items_img/cute.png">
                                            </div>
                                        </div>
                                        <div>
                                            <div id="fast">
                                                <img src="img/items_img/fast.png">
                                            </div>
                                        </div>
                                        <div>
                                            <div id="chess">
                                                <img src="img/items_img/chess.png">
                                            </div>
                                        </div>
                                        <div>
                                            <diV id="horse">
                                                <img src="img/items_img/horse.png">
                                            </diV>
                                        </div>
                                        <div>
                                            <div id="golden">
                                                <img src="golden/img/logo.png">
                                            </div>
                                        </div>
                                    </div>
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                </div>
                            </div>

                            <div class="item">
                                <div class="slider-body second-slider-body">
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                    <div class="items-inner">
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                        <div></div>
                                    </div>
                                </div>
                            </div>
                        </div>

                        <div class="footer-item">
                            <div class="items-inner">
                                <div>
                                    <div class="items_img" id="call">
                                        <img src="img/items_img/call.jpg">
                                    </div>
                                </div>
                                <div>
                                    <div class="items_img" id="contact">
                                        <img src="img/items_img/contact.png">
                                    </div>
                                </div>
                                <div>
                                    <div class="items_img" id="message">
                                        <img src="img/items_img/message.png">
                                    </div>
                                </div>
                                <div>
                                    <div class="items_img" id="setting">
                                        <img src="img/items_img/setting.jpg">
                                    </div>
                                </div>
                                <div>
                                    <div class="items_img" id="calc">
                                        <img src="img/items_img/Calculator.png">
                                    </div>
                                </div>
                            </div>
                        </div>
                    </div>
                    <div class="content-loader-div">
                        <div class="content-relative">
                            <div class="app-open-body">
                            </div>
                        </div>
                    </div>
                </div>
            </div>

            <div>
                <div class="navigation-button">
                    <div><i class="fal fa-bars"></i></div>
                    <div class="back"><i class="fal fa-square"></i></div>
                    <div class="back"><i class="fal fa-caret-left"></i></i></div>
                </div>
            </div>
        </div>  
    </div>

    <script src="js/jquery.min.js"></script>
    <script src="js/owl.carousel.min.js"></script>
    <script src="js/page.js"></script>
</body>
</html>