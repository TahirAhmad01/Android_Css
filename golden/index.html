<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<title>Golden</title>
	<link rel="icon" type="icon" href="img/logo.png">
	<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">	
	<link rel="stylesheet" type="text/css" href="css/animate.css">
	<link rel="stylesheet" type="text/css" href="css/lightbox.min.css">
	<link rel="stylesheet" type="text/css" href="style.css">
</head>
<body>
	<div class="loader-container">
        <div class="loader"></div>
	</div>
	
	<div class="go_top" id="top">
		<div></div>
		<ul>
			<li><i class="fas fa-arrow-alt-circle-up"></i></li>
			<li><p>Top</p></li>
		</ul>	
	</div>

	<header id="home">
		<div class="alc">
			<div id="menu" class="menu_bar">
				<div class="container">
					<div class="row">
						<div class="col-3">
							<div class="logo">
								<a href="#"><h2>Golden</h2></a>
							</div>
						</div>
						<div class="col-9">
							<div class="menu_icon_pos" id="menu_ico">
								<div class="menu_icon">
									<div class="bar1"></div>
									<div class="bar2"></div>
									<div class="bar3"></div>
								</div>
							</div>
							<div class="menu">
								<ul class="nav justify-content-end" id="nav">
									<li><a class="current" href="#home">home</a></li>
									<li><a href="#service">service</a></li>
									<li><a href="#portfolio">portfolio</a></li>
									<li><a href="#about">about</a></li>
									<li><a href="#skill">skill</a></li>
									<li><a href="#team">team</a></li>
									<li><a href="#contact">contact</a></li>
								</ul>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
	</header>

	<div class="head_item header">
		<div class="head_bg"></div>
		<div class="container">
			<div class="top_tag">
				<p>Welcome To Our Studio!</p>
				<h1>It’s nice to Meet you</h1>	
				<a href="#service">
					<button class="button"><text>tell me more</text></button>
				</a>
			</div>
		</div>
		</div>

	<section id="service">
		<div class="container">
			<div class="sec_head">
				<h2>service</h2>
				<p>Proin iaculis purus consequat sem cure. </p>
			</div>

			<div class="row">
				<div class="col-md-4">
					<div class="ser_con">
						<div class="ser_img">
							<img src="img/ico_1.png">
						</div>
						<h4>E-Commerce</h4>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt</p>
					</div>
				</div>

				<div class="col-md-4">
					<div class="ser_con">
						<div class="ser_img">
							<img src="img/ico_2.png">
						</div>
						<h4>Responsive Web</h4>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
					</div>
				</div>
				<div class="col-md-4">
					<div class="ser_con">
						<div class="ser_img">
							<img src="img/ico_3.png">
						</div>
						<h4>Web Security</h4>
						<p>Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt.</p>
					</div>
				</div>
			</div>
		</div>
	</section>

	<section id="portfolio">
		<div class="container">
			<div class="sec_head">
				<h2>Our portfolio</h2>
				<p>Proin iaculis purus consequat sem cure. </p>
			</div>

			<div class="button-group filter-button-group portfolio_btn">
				<button class="is-checked" data-filter="*">Show All</button>
				<button data-filter=".Branding">Branding</button>
				<button data-filter=".print">Print Design</button>
				<button data-filter=".mockUp">Product Mock-Up</button>
				<button data-filter=".design">Web Design</button>
			</div>

			<div class="row grid justify-content-between porfolio">
				<div class="col-lg-4 col-md-6 col-sm-12 grid-item Branding">
					<div class="prfl">
						<div class="img_pos">
							<a href="img/p1.png" data-lightbox=”gallery”>
								<img src="img/p1.png">
							</a>
						</div>
						<div class="prf_des">
							<h4>Ebony & Ivory</h4>
							<p>branding</p>
						</div>
					</div>
				</div>

				<div class="col-lg-4 col-md-6 col-sm-12 grid-item print">
					<div class="prfl">
						<div class="img_pos">
							<a href="img/p2.png" data-lightbox=”gallery”>
								<img src="img/p2.png">
							</a>
						</div>
						<div class="prf_des">
							<h4>Smart Stationary</h4>
							<p>print design</p>
						</div>
					</div>
				</div>

				<div class="col-lg-4 col-md-6 col-sm-12 grid-item print">
					<div class="prfl">
						<div class="img_pos">
							<a href="img/p3.png" data-lightbox=”gallery”>
								<img src="img/p3.png">
							</a>
						</div>
						<div class="prf_des">
							<h4>Clever Poster</h4>
							<p>print design</p>
						</div>
					</div>
				</div>

				<div class="col-lg-4 col-md-6 col-sm-12 grid-item mockUp">
					<div class="prfl">
						<div class="img_pos">
							<a href="img/p4.png" data-lightbox=”gallery”>
								<img src="img/p4.png">
							</a>
						</div>
						<div class="prf_des">
							<h4>Vinyl Record</h4>
							<p>Product Mock-Up</p>
						</div>
					</div>
				</div>

				<div class="col-lg-4 col-md-6 col-sm-12 grid-item design">
					<div class="prfl">
						<div class="img_pos">
							<a href="img/p5.png" data-lightbox=”gallery”>
								<img src="img/p5.png">
							</a>
						</div>
						<div class="prf_des">
							<h4>Treehouse Template</h4>
							<p>Web design</p>
						</div>
					</div>
				</div>

				<div class="col-lg-4 col-md-6 col-sm-12 grid-item Branding">
					<div class="prfl">
						<div class="img_pos">
							<a href="img/p6.png" data-lightbox=”gallery”>
								<img src="img/p6.png">
							</a>
						</div>
						<div class="prf_des">
							<h4>Burned Logo</h4>
							<p>branding</p>
						</div>
					</div>
				</div>
			</div>
		</div>
	</section>

	<section id="about" class="about">
		<div class="container">
			<div class="sec_head">
				<h2>About us</h2>
				<p>Proin iaculis purus consequat sem cure. </p>
			</div>

			<div class="about_con">
				<div class="about_text ab_left">
					<div class="ab_img ab_l">
						<img src="img/ab_1.png">
					</div>
					<div class="wow animate__fadeInLeft" data-wow-duration="2s">
						<h4>
							July 2010<br>
							Our Humble Beginnings
						</h4>
						<p>
							Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
						</p>
					</div>
				</div>

				<div class="about_text ab_right">
					<div class="ab_img ab_r">
						<img src="img/ab_2.png">
					</div>
					<div class="wow animate__fadeInRight" data-wow-duration="2s">
						<h4>
							January 2011<br>
							Facing Startup Battles

						</h4>
						<p>
							Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
						</p>
					</div>
				</div>


				<div class="about_text ab_left">
					<div class="ab_img ab_l">
						<img src="img/ab_3.png">
					</div>
					<div class="wow animate__fadeInLeft" data-wow-duration="2s">
						<h4>
							January 2011<br>
							Facing Startup Battles

						</h4>
						<p>
							Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
						</p>
					</div>
				</div>

				<div class="about_text ab_right">
					<div class="ab_img ab_r">
						<img src="img/ab_4.png">
					</div>
					<div class="wow animate__fadeInRight" data-wow-duration="2s">
						<h4>
							January 2011<br>
							Facing Startup Battles

						</h4>
						<p>
							Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit aenean rhoncus posuere odio in tincidunt. Proin iaculis purus consequat sem cure digni ssim. Donec porttitora entum suscipit.
						</p>
					</div>
				</div>
			</div>

			<div class="ab_btn">
				<a href="#">
					<button>Our<br> Story<br> Continues<br> ...</button>
				</a>
			</div>
		</div>
	</section>

	<section id="skill">
		<div class="container">
			<div class="sec_head">
				<h2>Skill</h2>
				<p>Proin iaculis purus consequat sem cure. </p>
			</div>

			<div class="skill_outside"  data-percent="90%">
				<div class="skill">

					<div class="skill_head">
						<label>html</label>
						<div></div>
					</div>

					<div class="skill_bar">
						<div class="wow animate__fadeInLeft" data-wow-duration="3s"></div>
					</div>
				</div>
			</div>

			<div class="skill_outside" data-percent="95%">
				<div class="skill">

					<div class="skill_head">
						<label>css</label>
						<div></div>
					</div>

					<div class="skill_bar">
						<div class="wow animate__fadeInLeft" data-wow-duration="3s"></div>
					</div>
				</div>
			</div>

			<div class="skill_outside" data-percent="15%">
				<div class="skill_head">
					<label>php</label>
					<div></div>
				</div>

				<div class="skill_bar">
					<div class="wow animate__fadeInLeft" data-wow-duration="3s"></div>
				</div>
			</div>

			<div class="skill_outside" data-percent="45%">
				<div class="skill_head">
					<label>javascript</label>
					<div></div>
				</div>

				<div class="skill_bar">
					<div class="wow animate__fadeInLeft" data-wow-duration="3s"></div>
				</div>
			</div>

			<div class="skill_outside" data-percent="55%">
				<div class="skill_head">
					<label>jquary</label>
					<div></div>
				</div>

				<div class="skill_bar">
					<div class="wow animate__fadeInLeft" data-wow-duration="4s"></div>
				</div>
			</div>

			<div class="skill_outside" data-percent="85%">
				<div class="skill_head">
					<label>psd to html</label>
					<div></div>
				</div>
				<div class="skill_bar">
					<div class="wow animate__fadeInLeft" data-wow-duration="4s"></div>
				</div>
			</div>

			<div class="skill_outside" data-percent="30%">
				<div class="skill_head">
					<label>logo design</label>
					<div></div>
				</div>
				<div class="skill_bar">
					<div class="wow animate__fadeInLeft" data-wow-duration="4s"></div>
				</div>
			</div>

		</div>
		
	</section>

	<section id="team">
		<div class="container">
			<div class="sec_head">
				<h2>Our amazing team</h2>
				<p>Proin iaculis purus consequat sem cure. </p>
			</div>
			<div class="team_img">
				<div class="team_con">
					<a href="img/1.jpg" data-lightbox=”tean”>
						<img src="img/1.jpg">
					</a>
					
					<div class="team_des">
						<h4>Kimberly Thompson</h4>
						<p>Marketer</p>
						<ul>
							<a href="#"><li><i class="fab fa-facebook-f"></i></li></a>
							<a href="#"><li><i class="fab fa-twitter"></i></li></a>
						</ul>
					</div>
				</div>

				<div class="team_con">
					<a href="img/2.jpg" data-lightbox=”tean”>
						<img src="img/2.jpg">
					</a>
					
					<div class="team_des">
						<h4>Rico Massimo</h4>
						<p>Coder</p>
						<ul>
							<a href="#"><li><i class="fab fa-facebook-f"></i></li></a>
							<a href="#"><li><i class="fab fa-twitter"></i></li></a>
						</ul>
					</div>
				</div>

				<div class="team_con">
					<a href="img/3.jpg" data-lightbox=”tean”>
						<img src="img/3.jpg">
					</a>
					
					<div class="team_des">
						<h4>Uku Mason</h4>
						<p>Graphic Designer</p>
						<ul>
							<a href="#"><li><i class="fab fa-facebook-f"></i></li></a>
							<a href="#"><li><i class="fab fa-twitter"></i></li></a>
						</ul>
					</div>
				</div>

			</div>

		</div>
	</section>

	<div class="section">
		<div class="container">
			<div class="spn_con">
				<div class="spn">
					<a href="#"><img src="img/v1.png"></a>
				</div>
				<div class="spn">
					<a href="#"><img src="img/v2.png"></a>
				</div>
				<div class="spn">
					<a href="#"><img src="img/v3.png"></a>
				</div>
				<div class="spn">
					<a href="#"><img src="img/v4.png"></a>
				</div>
			</div>
		</div>
	</div>

	<section id="contact" class="contact">
		<div class="c_over">
			<div class="container">
				<div class="sec_head">
					<h2>Contact us</h2>
					<p>Proin iaculis purus consequat sem cure. </p>
				</div>

				<form>
					<div class="c_form">
						<div class="c_input" >
							<input class="m_b" type="text" placeholder="Your Name">
							<input class="m_b wow animate__fadeInLeft" data-wow-duration="1s" type="text" placeholder="Youe E-mail">
							<input type="text" class="wow animate__fadeInLeft" data-wow-duration="2s" data-wow-duration="2s" placeholder="subject">
						</div>
						<div class="c_textarea wow animate__fadeInRight" data-wow-duration="3s">
							<textarea placeholder="message"></textarea>
						</div>
					</div>
					<div class="c_btn">
						<a href="#">
							<button class="button"><text>send message</text></button>
						</a>
					</div>
				</form>
			</div>
		</div>
	</section>

	<footer>
		<div class="container">
			<div class="row ftr" data-aos="new-animation">
				<div class="col-sm-5">
					<p>© Copyright 2020 <a href="http://nazara.rf.gd" target="_blank"> Nazara</a></p>
				</div>
				<div class="col-sm-7">
					<div class="social_ico">
						<ul>
							<a href="https://web.facebook.com/simanto.saikat.1/" target="_blank">
								<li><i class="fab fa-facebook-f"></i></li>
							</a>
							<a href="https://twitter.com/Simantosaikat01" target="_blank">
								<li><i class="fab fa-twitter"></i></li>
							</a>
							<a href="https://www.instagram.com/simanto.saikat234/" target="_blank">
								<li><i class="fab fa-instagram"></i></li>
							</a>
						</ul>
					</div>
				</div>
			</div>
		</div>
	</footer>

	<script src="https://kit.fontawesome.com/f70f4c2534.js" crossorigin="anonymous"></script>
	<script type="text/javascript" src="js/jquery-3.5.1.min.js"></script>
	<script type="text/javascript" src="js/isotope.pkgd.min.js"></script>
	<script type="text/javascript" src="js/wow.min.js"></script>
	<script type="text/javascript" src="js/lightbox.min.js"></script>
	<script type="text/javascript" src="js/jquery.nav.js"></script>
	<script type="text/javascript" src="js/page_js.js"></script>
</body>
</html>